sectionproperties/analysis/cross_section.py

```diff
@@ -1758,109 +1872,19 @@ class PlasticSection:
     :type c_bot: list[float, float]
     """
 
-    def __init__(self, geometry, materials, debug):
+    def __init__(
+        self, geom: Union[sections.Geometry, sections.CompoundGeometry], debug: bool
+    ):
         """Inits the PlasticSection class."""
-
-        # make a deepcopy of the geometry & materials so that we can modify it
-        self.geometry = copy.deepcopy(geometry)
-        self.materials = copy.deepcopy(materials)
+        self.geometry = geom.align_center()
+        self.geometry.compile_geometry()
         self.debug = debug
```

Comment:
is this actually used anymore given the way the section is no longer meshed at each iteration with shapely version? 

I could not find anywhere where it is utilised, so I assume it can be removed as an argument and the associate doc for the argument?

Reply:
Are you referring to `self.debug`? It is something that Robbie had included in the code for trouble shooting. When I redid the implementation, all of the methods that took a `debug` argument were replaced. So, yeah, we should definitely remove it :)

Reply:
Removed!

Reply:
Yeah was referring to self.debug, as didn't seem to be used anymore with the new methods of solving for plastic axis 

(github is a little unclear on what line was commented on sometimes to me as well!)